src/us/kbase/workspace/database/provenance/Event.java

```diff
@@ -0,0 +1,62 @@
+package us.kbase.workspace.database.provenance;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A class representing resource lifecycle events
+ */
+
+public enum Event {
+
+	ACCEPTED("accepted"),
+	AVAILABLE("available"),
+	COPYRIGHTED("copyrighted"),
+	COLLECTED("collected"),
+	CREATED("created"),
+	ISSUED("issued"),
+	SUBMITTED("submitted"),
+	UPDATED("updated"),
+	VALID("valid"),
+	WITHDRAWN("withdrawn"),
+	OTHER("other");
+
+	// mapping of PIDs and lowercaseNames to ContributorRole
+	private static final Map<String, Event> STRING_TO_EVENT_MAP = new HashMap<>();
+	static {
+		for (final Event e : Event.values()) {
+			STRING_TO_EVENT_MAP.put(e.getEventName(), e);
+		}
+	}
+
+	private final String eventName;
+
+	private Event(final String eventName) {
+		this.eventName = eventName;
+	}
+
+	/**
+	 * Get the lowercase name of this contributor role.
+	 *
+	 * @return the lowercaseName.
+	 */
```

Comment:
This doc seems off

Reply:
this was a really shoddy PR! Thank goodness for code review.

Reply:
:+1: